src/app.rs

```diff
@@ -133,7 +147,8 @@ where
             .nest_service("/uploads", ServeDir::new(&uploads_dir))
             .with_state(Context {
                 names_plural: self.names_plural,
-                uploads_dir,
+                editor_config: self.editor_config.clone(),
+                uploads_dir: uploads_dir.clone(),
```

Comment:
Why is clone needed here? (Why wasn't this the case before?)

Reply:
We potentially need `uploads_dir` below, to pass in as an extension.

Reply:
Although instead, maybe we can just access the context instead from the relevant method.